actions/views.py

```diff
@@ -45,13 +53,19 @@ def check_action_log(request, log_id):
     """
     View for API that request in Actions page to check the status of the action log.
     It will return the value that used in live updating the Action status.
+
+    This view can only access by superuser and staff.
     """
     action = get_object_or_404(ActionLog, id=log_id)
-    if action.status == 1 or action.status == 0:
-        duration = (timezone.now() - action.time_start).total_seconds()
-    elif action.status == 2:
-        duration = (action.time_finish - action.time_start).total_seconds()
-    else:
+    try:
+        if action.status == 1 or action.status == 0:
```

Comment:
Can you add some comments or docstring in this code to explain the status number?

Reply:
I've added it. Sorry for that.